### DISCUSSION TITLE
Обмен  в приложении green wallet

### DISCUSSION THREAD

**Discussion by cccc555** [Chia-Network/chia-blockchain#17235] [Support]
Здравствуйте. Отправил заявку на обмен в паре xch/usdt в приложении Green wallet? Прошло несколько часов, а заявка имела статут “ в процессе", сбросил приложение в смартфоне, установил все заново. Теперь в приложении баланс ноль, в spacescan транзакция отображена -ID транзакции 0xff44ec58d52520e91bd6bf677237452b354a7d2ad2f5e1c2d37b1012c85607a8 , usdt на адрес который я указал не пришли, что делать?

**Comment by BrandtH22** [comment]
@cccc555 , as a heads up we can generally provide more timely and thorough support in our discord server (https://discord.gg/chia).

The chain has seen high utilization leading to fee pressure so any transactions submitted without fees are taking lower priority than those which have fees. Many users have been able to resolve their pending fees by cancelling the previous transaction onchain and resubmitting it with a higher fee. 

Unfortunately the Green App is a third party wallet and you would need to contact their team to further troubleshoot issues with that system. 
If you are able to import your keys into the official chia desktop wallet then we will be able to assist further.

**Comment by cccc555** [comment]
Как связаться с командой разработчиков кошелька? 
--
Отправлено из Mail.ru для Android вторник, 09 января 2024г., 18:56 +03:00 от ClydeWallace22  ***@***.*** :

**Reply by BrandtH22** [reply]
The best place would be through the contact information from their website: https://green-app.io/

It looks like they have email, twitter, telegram, and discord as contact methods 👍

**Comment by BrandtH22** [comment]
If you are still experiencing the issues it is recommended to upgrade to the latest version of chia. Please download it from here: https://www.chia.net/downloads/

Since this issue has been open for a number of years without additional comments, we will be closing this ticket but if we have closed this ticket in error do not hesitate to reach out to us again with any followup questions, comments, or if the issue persists after an update.

```
Please note that upgrading major versions of chia generally requires the config.yaml to be recreated using these steps:
1. Stop chia
2. Rename the config or move it to a different directory
3. In CLI run chia init
4. Copy any custom settings from the old config to the new (plot drive directories, pool info, trusted node info, etc) - be careful here as yaml is very picky about formatting including those leading spaces
5. Start Chia
```

The best place to reach our support team is on Discord (https://discord.gg/chia) or by reopening this ticket.
